### DISCUSSION TITLE
Re3gistry instaces

### DISCUSSION THREAD

**Discussion by arantzaetxebarria** [ec-jrc/re3gistry#372] [Show and tell]
Dear community

To enhance the dynamism of the existing Re3gistry software instances status, we present a table that compiles the information we currently collect. We encourage users to utilize this thread for comments and to have a table as updated as possible.

Please help us to improve and feel free to contribute any related information. Thanks

Instance | Organisation | Country | Version used | Status
-- | -- | -- | -- | --
INSPIRE Registry | ​​European Commission​ | European Commission | 2.0 | In production
​​ENIPI​​​ Registry system | Ministry of the Environment of the Slovak Republic | Slovakia | 0.4 | In production
​​Effective Management of Spatial Data and Services (ESPUS)​ | Ministry of Environment of the Slovak Republic | Slovakia | 2.3.0 | In testing
​​NSDI registry​​​ | ​​Agency for Real Estate Cadastre​ (Republic of North Macedonia​) | Republic of North Macedonia | 0.x | Dismissed
​​Geosmartcity registry​ | European Commission | European Commission | [N/A] | Dismissed
​​ELISE Energy Pilot registry​ | European Commission | European Commission | 2.3.0 | In production
​​Austrian INSPIRE Registry​​​ | Austrian INSPIRE Registry​ | Austria | 2.3.0 | Online
Italian INSPIRE Registry | Italia INSPIRE Registry | Italia | 2.0.0 | Online
​​Finland INSPIRE Registry | NLS National Land Survey of Finland | Finland | 1.3.1 | Online
​​Spanish INSPIRE Registry | Spain INSPIRE Registry (IDEE) | Spain | 1.3.1 | Online
​​Czech INSPIRE Registry | ​​Czech Office for Surveying, Mapping and Cadastre​ | Czech Republic | 2.0 | Online
​​GDI-DE Registry​ | Federal Agency for Cartography and Geodesy​ | Germany | 1.x | Online
Private - Minerva intelligence​ | Minerva Intelligence Inc.​​ | [N/A] | [N/A] | Dismissed
Wallonie GeoRegistry | ​​​​Service public ​of​ Wallonie​​​​​ (Belgium) | Belgium | 1.x | Online
Re3gistry 2 | Metaregister | European Commission | 2.0 | [N/A]
Geographic Item Registration System | Spatial Data Infrastructure for the Capital District | Republic of Colombia | 2.0.0 | Online
Epsilon Italia Registry | Epsilon Italia | Italia | 2.0.0 | Online
Geonovum Re3gistry2 | Geonovum | Netherlands | 2.0.0 | Online

**Comment by weinKonrad** [comment]
Will the list be updated and continued? The same applies to the list at https://interoperable-europe.ec.europa.eu/collection/are3na/solution/re3gistry/re3gistry-reusers 
It seems that the status of some entries is no longer correct.
